code/modules/mob/living/silicon/robot/robot_defines.dm

```diff
@@ -202,10 +202,26 @@
 	cell = /obj/item/stock_parts/cell/hyper
 	radio = /obj/item/radio/borg/syndicate
 
+/mob/living/silicon/robot/model/syndicate/Initialize(mapload)
+	laws = new /datum/ai_laws/syndicate_override()
+	laws.associate(src)
+	. = ..()
+	addtimer(CALLBACK(src, PROC_REF(show_playstyle)), 5)
+
+/mob/living/silicon/robot/model/syndicate/create_modularInterface()
+	if(!modularInterface)
```

Comment:
```suggestion
	if(isnull(modularInterface))
```

this works right?

Reply:
I didn't write this, just moved it, so I'll leave it

Reply:
that'sf ine